web-api/src/database-types.ts

```diff
@@ -1,310 +1,447 @@
 import { ColumnType, Insertable, Selectable, Updateable } from 'kysely';
 import { MinuteSheet } from '@shared/business/entities/trialSessionMinutes/MinuteSheet';
+import { DW_MINUTE_SHEET_COLUMNS } from '@web-api/persistence/postgres/minuteSheets/mapper';
 
-export type DatabaseTableName = keyof Database;
-
-export interface Database {
-  dwCase: CaseTable;
-  dwMinuteSheet: MinuteSheetTable;
-  dwCaseCorrespondence: CaseCorrespondenceTable;
-  dwCaseDeadline: CaseDeadlineTable;
-  dwCaseStatistic: CaseStatisticTable;
-  dwCaseStatusUpdate: CaseStatusUpdateTable;
-  dwCaseWorksheet: CaseWorksheetTable;
-  dwDocketEntry: DocketEntryTable;
-  dwMessage: MessageTable;
-  dwPetitionerOnCase: PetitionerOnCaseTable;
-  dwStatisticPenalty: StatisticPenaltyTable;
-  dwUserCaseNote: UserCaseNoteTable;
-  dwWorkItem: WorkItemTable;
-}
-
-export interface MessageTable {
-  attachments?: ColumnType<{ documentId: string }[], string, string>;
-  completedAt?: Date;
-  completedBy?: string;
-  completedBySection?: string;
-  completedByUserId?: string;
-  completedMessage?: string;
-  createdAt: Date;
-  docketNumber: string;
-  from: string;
-  fromSection: string;
-  fromUserId: string;
-  isCompleted: boolean;
-  isRead: boolean;
-  isRepliedTo: boolean;
-  message: string;
-  messageId: string;
-  parentMessageId: string;
-  subject: string;
-  to: string;
-  toSection: string;
-  toUserId: string;
-}
+const messageTableDefinition = {
```

Comment:
The general idea: define dummy object with type information and dynamically generate stuff from it to avoid having to re-specify data like, e.g., column names.